src/modules/colorPicker/ColorPickerUI/Controls/ColorFormatControl.xaml

```diff
@@ -45,17 +44,16 @@
             <Button x:Name="CopyToClipboardButton"
                     ToolTipService.ToolTip="{x:Static p:Resources.Copy_to_clipboard}"
                     Background="{DynamicResource ColorControlBackgroundBrush}"
+                    FontSize="16"
+                    FontFamily="{StaticResource SymbolThemeFontFamily}"
```

Comment:
Was this tested? IIRC this doesn't work in WPF by default? Might be that ModernWPF has added this as a resource though.. that'd be nice.

Reply:
https://github.com/Kinnara/ModernWpf/blob/6580ce2cb3af362b1db8518d04b605c5a295968b/ModernWpf/ControlsResources.xaml#L51

Seems like `SymbolThemeFontFamily` was added to ModernWPF.. but hardcoded to Segoe MDL2 Assets so no visual changes there.

Reply:
You're right: I **does** work. And running on my Surface with Windows 11 shows your second point. Might be an argument to move away from (modern)WPF. I would keep the static resource and switch to WinUI 3 in the future.